app/src/main/java/com/vuongnq14798/vuongmp3/ui/home/HomeFragment.kt

```diff
@@ -90,7 +63,7 @@ class HomeFragment : BaseFragment(),
         startActivity(context?.let { DetailGenreActivity.getIntent(it, genre) })
 
     override fun onTrackClicked(track: Track) {
-        mediaPlayerService.changeTrack(track)
-        context?.let { startActivity(PlayMusicActivity.getIntent(it)) }
+        (activity as MainActivity).getMediaPlayerService()?.let { it.changeTrack(track) }
```

Comment:
```suggestion
        (activity as MainActivity).getMediaPlayerService()?.changeTrack(track) 
```
Viết đc thế này mà nhỉ